### ISSUE TITLE
Schematic Doesn't Exist

### ISSUE THREAD

**Issue by MeAngel** [IntellectualSites/PlotSquared#592]
When we type /plot createroadschematic we get the message ingame that the schematic was pasted and to view the console.

When we view the console, there is this message:

[Server] INFO plugins/PlotSquared/schematics/GEN_ROAD_SCHEMATIC/13thGate/plot.schematic doesn't exist

And then of course when we get to the point where it's time to run /plot regenallroads  the results are very disappointing. 

I guess the first thing to address is the error message from console (above) and how it conflicts with the ingame message that the file was created.

Also, I'm wondering if there have been any requests / plans to "undo" or revert your map after you've run: /plot debugroadregen

The instructions for installation say only this about that step:

If all is well, you can begin regenerating the roads in the entire map.

But what if all is not well? What are the next steps?

Thanks!

**Comment by TobyMellor** [comment]
The schematic system for /plot schematic is very confusing too

**Comment by boy0001** [comment]
`plot.schematic` is for inside the plot. The sideroad and intersection were created. Also, it's not an error message, it's an info message.

What to look for is if you go to another plot and run `/plot debugroadregen`, if that results in a road resembling what you want, then it worked.

If it doesn't work out, you can redo the road around the plot and set a new schematic. If you think it's a bug with the plugin, could you please provide more information?

**Comment by MeAngel** [comment]
It seems to have a bug unless I'm misunderstanding the instructions.  We had a road schematic that surrounded the entire plot originally. It included a "roof", a structure at the intersections, etc. 

After having it fail repeatedly to work correctly we simplified it to one intersection and a north road and a west road - is this where the issue is? I've attached a screenshot showing where the roads were and where I stood (if that's important) when I create the road schematic. 

Is that the issue? Do we have the wrong intersection and roads?

![generateroadschem](https://cloud.githubusercontent.com/assets/12853129/9606734/1937922a-507a-11e5-88e6-a71ed5be0a8a.jpg)

**Comment by MeAngel** [comment]
I am stubborn and I do not want to settle for a plain single stone road for my world. Is there anyone that I can pay to get this thing to work? It's possible, I see screenshots of it so I know that somewhere out there is someone who can actually help me.

I'm beyond frustrated with failed attempt after failed attempt. Nothing I do seems to make a difference. 

If there is anyone out there who can take my road and intersection schematics and get the roads to turn out as they should I will pay them.

**Comment by boy0001** [comment]
Those are the correct sections. What do you mean by it failing though, you have never specified what actually happens?

**Comment by MeAngel** [comment]
It's in the video but what happens is that when the road starts generating there are incomplete sections, gaps at intersections, at one point the black glass turned to white glass. 

But to sum it up it doesn't generate across the whole map, the sections that do generate usually stop a row or two at the intersection, or in some cases about 10 rows from it. 

It just will not paste properly. If you look in the video clip, which was taken in a fresh world with no roads anywhere but the intersection and the roads we'd placed, once we run the command from console the roads that start generating had a lot of issues. Our plots are 150 wide, I don't know if that is an issue. 

But it seemed that the few times we tested, and then adjusted and ran the schematic again, it just got messier and messier so we skipped the test part so we could be sure that what we were seeing was being generated by the command and not our botched test results.

Does this help:  https://www.youtube.com/watch?v=gfJvYpigueI
